tests/integrations/config/test-custom.toml

```diff
@@ -150,6 +150,8 @@ info-log-roll-time = "12s"
 info-log-keep-log-file-num = 1000
 info-log-dir = "/var"
 rate-bytes-per-sec = "1KB"
+rate-limiter-mod = 3
```

Comment:
Better to not use default value in test.

Reply:
Ok, I will fix it. Thanks.